azure-pipelines-nightly.yml

```
@@ -558,29 +558,34 @@ jobs:
       TargetPlatform: "esp32"
+      IDF_TAG: "v5.2.3"
 
     steps:
       - checkout: self
-      - checkout: esp32_idf
-        fetchDepth: 1
+        path: s/nf-interpreter
 
+      - template: azure-pipelines-templates/checkout-idf.yml
+        parameters:
+          repoDirectory: '$(Agent.BuildDirectory)/s'      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)